CHANGELOG.md

```diff
@@ -7,21 +7,35 @@
 [Full log](https://github.com/sofa-framework/sofa/compare/v20.06...HEAD)
 
 
-### Deprecated
+### Breaking
 
+- Ongoing [SOFA-NG project](https://github.com/sofa-framework/sofa/issues/1527) generate several breaking changes due to modularization. It currently targets SOFA packages (SofaMisc, [SofaGeneral](https://github.com/sofa-framework/sofa/issues/1529) and soon SofaCommon), as well as SofaKernel (SofaBase\*, SofaCore, SofaHelper, SofaSimulation). Follow the [progress of the project online](https://github.com/sofa-framework/sofa/projects/9#card-47075575).
+  - [Modularization] SofaMisc* (Revival) [#1520](https://github.com/sofa-framework/sofa/pull/1520)
+- Ongoing [Uniform type for indices and sizes](https://github.com/sofa-framework/sofa/issues/1539). Following PR have been performed and update the *size_type* and *index_type* in SOFA:
+  - [All] Uniform size type [#1515](https://github.com/sofa-framework/sofa/pull/1515)
+  - [All] Change index_type from size_t to uint [#1514](https://github.com/sofa-framework/sofa/pull/1514)
+  - [All] Standardize index type for Vector/Matrix templates [#1453](https://github.com/sofa-framework/sofa/pull/1453)
+- Ongoing [cleaning of the Link implementation](https://github.com/sofa-framework/sofa/issues/1526)
```

Comment:
I would break the breaking into 2 parts here:
- The big big breaking (sofa ng + size_t) and the usual breaking